content/home/milestones/milestones/milestone3.md

```diff
@@ -1,6 +1,6 @@
 ---
 title : "milestone"
 heading : "Pioneer 1: V3 Network Beta"
-status : isCurrent
+status : isFuture
```

Comment:
why is this isFuture? Shouldn't it be isPast or something?

Reply:
I think this is the class that was given to both past and future milestones

Reply:
ah ok haha